### ISSUE TITLE
IconsaxBulk not working properly

### ISSUE THREAD

**Issue by gideonoyekale** [Mastersam07/fIconsax#7]
IconsaxBulk is not working properly, most icons don't even show at all with IconsaxBulk. Some show but with whitespaces where the part with other color should be shown

**Comment by Mastersam07** [comment]
Hi @gideonoyekale thanks for raising an issue.

I have noticed this behaviour and traced the issue down to the font rendering the icons. Will compare with the iconsax design library to effect the changes.

For now, you might want to use another variant.

**Comment by gideonoyekale** [comment]
> Hi @gideonoyekale thanks for raising an issue.
> 
> I have noticed this behaviour and traced the issue down to the font rendering the icons. Will compare with the iconsax design library to effect the changes.
> 
> For now, you might want to use another variant.

Thanks for your response.

The problem seem to be with the fact that the Bulk variant icons are two coloured. I have also tried to convert the svgs to font but got the same result. 

This might help you further in resolving this as I don't think Iconsax themselves have icons for each of these variants

**Comment by gideonoyekale** [comment]
https://stackoverflow.com/questions/71939718/icon-font-with-multi-color-svg-icons

You might find this helpful too

**Comment by gideonoyekale** [comment]
@Mastersam07 

https://stackoverflow.com/questions/71939718/icon-font-with-multi-color-svg-icons

You might find this helpful too

**Comment by Mastersam07** [comment]
Yes @gideonoyekale. Which is why it affect bulk icons and duotone(two tone) icons. I have some idea off the top of my head by haven't given it a try.

Will make a patch and update this issue if it fixes it.

Thanks.

**Comment by gideonoyekale** [comment]
> Yes @gideonoyekale. Which is why it affect bulk icons and duotone(two tone) icons. I have some idea off the top of my head by haven't given it a try.
> 
> Will make a patch and update this issue if it fixes it.
> 
> Thanks.

You are welcome. Thanks for your responses too!

**Comment by Mastersam07** [comment]
Perhaps its time to revisit this issue.
